packages/picasso.js/src/core/chart/component-collection.js

```diff
@@ -105,15 +105,16 @@ function collectionFn({ createComponent }) {
           // Component is added
           return createComp(comp);
         }
-
+        if (comp.rendererSettings && typeof component.instance.renderer().settings === 'function') {
```

Comment:
This is to fix the issue where component.instance.renderer().settings is not updated after data updated e.g. because of selection.